src/index.css

```diff
@@ -0,0 +1,169 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.App {
+  background-color: rgb(81, 130, 221);
+  width: 90%;
+  margin: 1rem auto;
+  border-radius: 15px;
+  box-shadow: 10px 10px 5px 0 rgba(0, 0, 0, 0.75);
+  -webkit-box-shadow: 10px 10px 5px 0 rgba(0, 0, 0, 0.75);
+  -moz-box-shadow: 10px 10px 5px 0 rgba(0, 0, 0, 0.75);
+}
+
+.header-container {
+  display: flex;
+  padding: 1rem;
+  margin-bottom: 1rem;
+}
+
+.angle-Left {
+  display: flex;
+  justify-content: flex-start;
+  width: 80%;
+}
+
+.left,
+.mic,
+.ger,
+h1,
+h3 {
+  color: #e9e2e2;
+}
+
+.cont {
+  width: 20%;
+  display: flex;
+  justify-content: right;
+  gap: 1rem;
+}
+
+.hero {
+  background-color: rgb(65, 102, 174);
+  width: 90%;
+  height: 35vh;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+  padding: 1rem;
+}
+
+h1,
+h3 {
+  text-align: center;
+  margin: 1rem;
+}
+
+span {
+  padding: 1rem;
+}
+
+.search {
+  display: block;
+  width: 50%;
+  padding: 0.5rem;
+  margin: 1rem auto;
+}
+
+.cards {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 5px;
```

Comment:
- [ ] you have done fantastic work on your designs, however, there is a minute change to make, if you look closely to the given design, you will see that there is no space or gap between cards, but yours have spaces tho tiny between cards, kindly remove it
your design :point_down: 
![Screenshot from 2023-09-08 11-32-14](https://github.com/MostafaShahat2021/reactCapstoneProject/assets/66097371/6b0bd62a-d933-47ac-913e-042f521a2f3f)
given design :point_down: 
![Screenshot from 2023-04-28 11-04-25](https://github.com/MostafaShahat2021/reactCapstoneProject/assets/66097371/19cc40e9-5475-4dd3-a380-6ae226bef1f5)

Reply:
Thanks , I'll fix it 👍